test/specs/scale.time.tests.js

```diff
@@ -127,7 +127,9 @@ describe('Time scale tests', function() {
 			var ticks = getTicksLabels(scale);
 
 			// `bounds === 'data'`: first and last ticks removed since outside the data range
-			expect(ticks).toEqual(['Jan 2', 'Jan 3', 'Jan 4', 'Jan 5', 'Jan 6', 'Jan 7', 'Jan 8', 'Jan 9', 'Jan 10']);
+			expect(ticks.length).toEqual(217);
+			expect(ticks[0]).toEqual('12PM');
```

Comment:
Why does a time span of 10 days yield `hour` unit? Seems wrong.

Reply:
The scale generates as many ticks as can fit using the smallest unit possible. The time span alone does not determine the unit as the scale width must also be taken into account. If the scale is sufficiently wide then there's room to generate hour labels between each day